bitcoin/pullpush.c

```diff
@@ -80,7 +80,7 @@ u64 pull_varint(const u8 **cursor, size_t *max)
 
 u32 pull_le32(const u8 **cursor, size_t *max)
 {
-	le32 ret;
+	le32 ret = 0;
```

Comment:
I think unnecessary? If `pull` succeeds the entire variable gets overwritten.  Similar `pull_le64`.

Reply:
Even if it *fails* it memsets the whole thing to zero, for safety.